py/persistentcode.c

```diff
@@ -404,14 +404,14 @@ STATIC mp_raw_code_t *load_raw_code(mp_reader_t *reader, qstr_window_t *qw) {
         ip2[2] = source_file; ip2[3] = source_file >> 8;
     }
 
+    // Number of entries in constant table
+    size_t n_obj = read_uint(reader, NULL);
+    size_t n_raw_code = read_uint(reader, NULL);
```

Comment:
This is not going to work: if `kind == MP_CODE_NATIVE_ASM` then there should be nothing read from the reader (and see that Travis build fails).